.dev/docker/default.conf

```diff
@@ -0,0 +1,51 @@
+# If using Ubuntu this file should be placed in:
+# 			/etc/nginx/sites-available/
+#
+# If using CentOS this file should be placed in:
+#				 /etc/nginx/conf.d/
+#
+server {
+    listen 80;
+    server_name _;
+    
+    root /app/public;
+    index index.html index.htm index.php;
+    charset utf-8;
+
```

Comment:
I would add the most basic security headers:
```
    add_header Referrer-Policy "strict-origin";
    add_header X-Xss-Protection "1; mode=block;" always;
    add_header X-Frame-Options "DENY";
    add_header X-Content-Type-Options nosniff;
```

SecurityHeaders.com is a good source. Here are two examples:
- https://securityheaders.com/?q=pterodactyl.io&hide=on&followRedirects=on
- https://securityheaders.com/?q=scotthelme.co.uk&hide=on&followRedirects=on